File: PythonCode/1043.py
#Questão: Triângulo
a,b,c = map(float, input().split())

if (a+b)>c and (b+c)>a and (c+a)>b:
    P = a+ b+c
    print("Perimetro =",P)
else:
    A = (((a + b) * c)/2)
    print("Area =",A)

File: C++Code/1043.cpp
//Questão: Triângulo
#include <iostream>
#include <iomanip>

using namespace std;

int main(){
    double a,b,c,P,A;
    cin >> a >> b >> c;
    if((a+b)>c && (b+c)>a && (c+a)>b){
        P = a + b + c;
        cout <<fixed<<setprecision(1)<< "Perimetro = "<<P<<endl;
    }
    else
    {
        A = ((a+b)*c)/2;
        cout <<fixed<<setprecision(1)<<  "Area = "<<A<<endl;
    }
    return 0;
}